src/test/java/hudson/plugins/promoted_builds/PromotedBuildActionTest.java

```
@@ -48,9 +49,11 @@ public void testDeletedPromotionProcess() throws Exception {
             } 
             HtmlImage img = (HtmlImage)candidate;
             try {
-                img.getHeight();
+                assertEquals("Failed to load " + img.getSrcAttribute(),
+                        200,
+                        img.getWebResponse(true).getStatusCode());
             } catch (IOException e) {
-                throw new IOException2("Failed to load "+img.getSrcAttribute(),e);
+                throw new AssertionError("Failed to load " + img.getSrcAttribute());
```

Or just let it be thrown.